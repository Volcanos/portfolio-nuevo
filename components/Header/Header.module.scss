.headerWrapper {
  align-items: center;
  border-bottom: 1px solid #e8e8e8;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  margin: 0 auto;
  padding: 0 0 60px 0;
  width: 100%;
}
