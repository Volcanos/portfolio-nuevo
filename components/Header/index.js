import Nav from '../Nav';
import Logo from '../Logo';
import styles from './Header.module.scss';

const Header = () => {
    return (
        <div className={styles.headerWrapper}>
            <Logo />
            <Nav />
        </div>
    )
}
export default Header;
