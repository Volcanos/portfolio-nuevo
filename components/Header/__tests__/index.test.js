import React from 'react'
import { mount, render } from 'enzyme';
import 'jest-styled-components'
import Header from '../index';

describe('Header', function () {
    it('snapshot not change', function () {
        const tree = render(<Header />);
        expect(tree).toMatchSnapshot();
    });
});