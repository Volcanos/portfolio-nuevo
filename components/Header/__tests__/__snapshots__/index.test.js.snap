// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot not change 1`] = `
.c4 {
  font-size: 1rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  position: relative;
}

.c4:hover {
  color: #6f6f6f;
  cursor: pointer;
}

.c4:hover:before,
.c4:hover:after {
  width: 50%;
  opacity: 1;
}

.c4:before,
.c4:after {
  content: "";
  position: absolute;
  bottom: 0px;
  width: 0px;
  height: 1px;
  margin: 2px 0 0;
  -webkit-transition: all 0.2s ease-in-out;
  transition: all 0.2s ease-in-out;
  -webkit-transition-duration: 0.65s;
  transition-duration: 0.65s;
  opacity: 0;
  background-color: #6f6f6f;
}

.c4:before {
  left: 50%;
}

.c4:after {
  right: 50%;
}

.c2 {
  list-style: none;
  margin: 0;
  padding: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c3 {
  margin-left: 1em;
}

.c5 {
  display: inline-block;
  position: relative;
}

.c5:hover {
  color: #6f6f6f;
  cursor: pointer;
}

.c5:hover:before,
.c5:hover:after {
  width: 50%;
  opacity: 1;
}

.c5:before,
.c5:after {
  content: "";
  position: absolute;
  bottom: 0px;
  width: 0px;
  height: 1px;
  margin: 2px 0 0;
  -webkit-transition: all 0.2s ease-in-out;
  transition: all 0.2s ease-in-out;
  -webkit-transition-duration: 0.65s;
  transition-duration: 0.65s;
  opacity: 0;
  background-color: #6f6f6f;
}

.c5:before {
  left: 50%;
}

.c5:after {
  right: 50%;
}

.c1 {
  font-size: 1.6rem;
  margin-bottom: 2rem;
}

.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-bottom: 1px solid #e8e8e8;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin: 0 auto;
  padding: 0 0 60px 0;
  width: 100%;
}

<header
  class="c0"
>
  <h1
    class="c1"
  >
    Borja Gracia Mart√≠n
  </h1>
  <nav>
    <ul
      class="c2"
    >
      <li
        class="c3"
      >
        <a
          class="c4"
        >
          Home
        </a>
      </li>
      <li
        class="c3"
      >
        <a
          class="c4"
        >
          Portfolio
        </a>
      </li>
      <li
        class="c3"
      >
        <a
          class="c4"
        >
          Experiencia
        </a>
      </li>
      <li
        class="c3"
      >
        <a
          class="c5"
        >
          Contacto
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
